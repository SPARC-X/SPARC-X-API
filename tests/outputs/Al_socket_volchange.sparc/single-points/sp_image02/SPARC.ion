# Ion File Generated by SPARC ASE Calculator
# 
# ASE-SORT:
# 0 1 2 3
# END ASE-SORT

ATOM_TYPE: Al
N_TYPE_ATOM: 4
PSEUDO_POT: 13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
COORD:
-0.08033093562375 -0.01084554712961 -0.41175620833334
0.31616558520924 3.55754090072064 3.74098077011621
4.00016074444072 -0.24003185000843 3.69930683823330
3.72801621818795 4.00952321866431 0.44182782750226

