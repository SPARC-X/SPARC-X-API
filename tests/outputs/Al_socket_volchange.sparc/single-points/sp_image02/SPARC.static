***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of Al:
      0.9897096828       0.9986106956       0.9472544174
      0.0405005138       0.4557176399       0.4792161144
      0.5124168251       0.9692521460       0.4738777230
      0.4775553675       0.5136161492       0.0565977287
Total free energy (Ha): -9.049946219980990E+00
Atomic forces (Ha/Bohr):
  3.4983196881E-03  -2.5771700762E-03  -3.8304353125E-03
 -1.3648434921E-02   4.3201426170E-03   5.5532264248E-03
  2.2414549146E-03   5.1274299183E-03   4.2342097525E-03
  7.9086603180E-03  -6.8704024590E-03  -5.9570008648E-03
Stress (GPa): 
  5.3310222375E+00   5.5148480239E-01   1.4621470847E+00 
  5.5148480239E-01   8.2658443311E-01  -8.1726042162E-01 
  1.4621470847E+00  -8.1726042162E-01  -3.3490689483E+01
