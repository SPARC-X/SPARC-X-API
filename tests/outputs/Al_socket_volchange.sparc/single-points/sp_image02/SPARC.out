***************************************************************************
*                       SPARC (version Oct 31, 2023)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Thu Jan 18 08:03:28 2024                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
LATVEC_SCALE: 1 1 1 
LATVEC:
7.806458625832350 0.000000000000000 0.000000000000000 
0.000000000000000 7.806458625832350 0.000000000000000 
0.000000000000000 0.000000000000000 7.806458625832350 
FD_GRID: 15 15 15
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.000950043469
EXCHANGE_CORRELATION: GGA_PBE
NSTATES: 12
CHEB_DEGREE: 17
CHEFSI_BOUND_FLAG: 0
CALC_STRESS: 1
MAXIT_SCF: 100
MINIT_SCF: 2
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-03
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-05
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-06
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 2.71E-04
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0
MIXING_PARAMETER: 1
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
NUM_CHEFSI: 1
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_ENERGY_DENSITY: 0
OUTPUT_FILE: SPARC
***************************************************************************
                                Cell                                       
***************************************************************************
Lattice vectors (Bohr):
7.806458625832350 0.000000000000000 0.000000000000000 
0.000000000000000 7.806458625832350 0.000000000000000 
0.000000000000000 0.000000000000000 7.806458625832350 
Volume: 4.7573180476E+02 (Bohr^3)
Density: 2.2686344054E-01 (amu/Bohr^3), 2.5422029172E+00 (g/cc)
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 2
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 1 1 2
EIG_SERIAL_MAXNS: 1500
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  2
Mesh spacing                       :  0.520431 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  SPARC.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  Al 3
Pseudopotential                    :  13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
Atomic mass                        :  26.9815385
Pseudocharge radii of atom type 1  :  7.29 7.29 7.29 (x, y, z dir)
Number of atoms of type 1          :  4
Estimated total memory usage       :  3.01 MB
Estimated memory per processor     :  1.50 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.2652342235E+00        2.212E-01        0.018
2            -2.2655998909E+00        2.553E-01        0.007
3            -2.2631429501E+00        1.349E-01        0.006
4            -2.2625606677E+00        3.863E-02        0.007
5            -2.2624915473E+00        1.621E-02        0.006
6            -2.2624876736E+00        6.655E-03        0.006
7            -2.2624862380E+00        1.743E-03        0.006
8            -2.2624840416E+00        1.633E-03        0.005
9            -2.2624865550E+00        6.575E-04        0.006
Total number of SCF: 9     
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2624865550E+00 (Ha/atom)
Total free energy                  : -9.0499462200E+00 (Ha)
Band structure energy              :  4.0506092524E-01 (Ha)
Exchange correlation energy        : -4.3182773432E+00 (Ha)
Self and correction energy         : -1.2498661550E+01 (Ha)
-Entropy*kb*T                      : -2.5075733855E-03 (Ha)
Fermi level                        :  1.1558592504E-01 (Ha)
RMS force                          :  1.0054081192E-02 (Ha/Bohr)
Maximum force                      :  1.5355185826E-02 (Ha/Bohr)
Time for force calculation         :  0.013 (sec)
Pressure                           :  9.1110276040E+00 (GPa)
Maximum stress                     :  3.3490689483E+01 (GPa)
Time for stress calculation        :  0.024 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  0.178 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*               List of contributors: See the documentation               *
*         Citation: See README.md or the documentation for details        *
*  Acknowledgements: U.S. DOE SC (DE-SC0019410), U.S. DOE NNSA (ASC)      *
*      {Preliminary developments: U.S. NSF (1333500,1663244,1553212)}     *
***************************************************************************
                                                                           
